lib/__tests__/index-test.js

```diff
@@ -353,7 +353,7 @@ describe('labels', () => {
 
   it('warns if an image without alt is the only content', () => {
     expectWarning(assertions.render.NO_LABEL.msg, () => {
-      <button><img src="#" alt=""/></button>;
+      <button><img src="#"/></button>;
```

Comment:
This markup would be an accessibility issue (unlabeled form control), wouldn't it?

The next test, `'does not warn if an image without alt is accompanied by text'`, shows an example of an empty `alt` attribute with label text, which is valid.